lib/entities/release.ts

```diff
@@ -80,12 +81,24 @@ export type ReleaseSysProps = {
   updatedAt: ISO8601Timestamp
   lastAction?: Link<'ReleaseAction'>
 }
-
+export type ReleasesReferenceFilters = ScheduledActionReferenceFilters
+
+export type MetadataAggeratedRootProps = {
```

Comment:
I would prefer it as `ReleaseMetdata` and not link this to Aggregated Root.

Reply:
okie